vdev/src/git.rs

```diff
@@ -9,6 +9,30 @@ pub fn current_branch() -> Result<String> {
     Ok(output.trim_end().to_string())
 }
 
+pub fn checkout_branch(branch_name: &str) -> Result<()> {
```

Comment:
I think the naming of these functions is a little unclear. Since this one function has a dual role, perhaps call this `checkout_or_create_branch`.

Reply:
I have implemented the changes